tools/topology/topology2/cavs/CMakeLists.txt

```diff
@@ -4,12 +4,6 @@
 set(TPLGS
 # CAVS HDMI only topology with passthrough pipelines
 "cavs-passthrough-hdmi\;cavs-passthrough-hdmi"
-# CAVS HDA topology with passthrough pipelines
```

Comment:
I thought passthrough pipelines could not work? Why keep them for HDMI?

Reply:
its passthrough but with separate FE and BE pipelines

Reply:
ah ok. makes sense.